src/bvals/cc/bvals_cc.cpp

```diff
@@ -132,12 +133,13 @@ int CellCenteredBoundaryVariable::LoadBoundaryBufferSameLevel(ParArray1D<Real> &
 
   IndexDomain interior = IndexDomain::interior;
   const IndexShape &cellbounds = pmb->cellbounds;
-  si = (nb.ni.ox1 > 0) ? (cellbounds.ie(interior) - NGHOST + 1) : cellbounds.is(interior);
-  ei = (nb.ni.ox1 < 0) ? (cellbounds.is(interior) + NGHOST - 1) : cellbounds.ie(interior);
-  sj = (nb.ni.ox2 > 0) ? (cellbounds.je(interior) - NGHOST + 1) : cellbounds.js(interior);
-  ej = (nb.ni.ox2 < 0) ? (cellbounds.js(interior) + NGHOST - 1) : cellbounds.je(interior);
-  sk = (nb.ni.ox3 > 0) ? (cellbounds.ke(interior) - NGHOST + 1) : cellbounds.ks(interior);
-  ek = (nb.ni.ox3 < 0) ? (cellbounds.ks(interior) + NGHOST - 1) : cellbounds.ke(interior);
+
+  cell_centered_bvars::CalcIndicesLoadSame(nb.ni.ox1, si, ei,
+                                           cellbounds.GetBoundsI(interior));
+  cell_centered_bvars::CalcIndicesLoadSame(nb.ni.ox2, sj, ej,
+                                           cellbounds.GetBoundsJ(interior));
+  cell_centered_bvars::CalcIndicesLoadSame(nb.ni.ox3, sk, ek,
```

Comment:
Are you referring to adding more documentation to the code or in the comment here?
For the former, the functions already contain brief docstrings. Is more info desired?
For the latter, I followed the existing naming convection, i.e., load = from var to buffer, and set = from buffer to var.
It also took me some time to remember this, so I'm open for a broader renaming of that capability if we have a quorum for that (it will touch quite a bit of machinery).

Reply:
Ah, I actually meant here just so I understood. ;)

These names make sense and aren't so important. I think it's fine to keep them. But recording that in the developers guide might be useful.